main/provider/helpers.ts

```diff
@@ -68,6 +68,13 @@ export function feeTotalOverMax (rawTx: TransactionData, maxTotalFee: number) {
   return totalFee > maxTotalFee
 }
 
+function parseValue(value?: string) {
+  const parsedHex = parseInt(value!!, 16)
```

Comment:
I don't think `value!!` works?  Not sure if you meant to use `!!value` though that doesn't work either as it's just effectively casting to boolean

Reply:
Ok, I see - would prefer to avoid use of non-null assertion though, just like using `any` it should be avoided if possible:

https://medium.com/@tar.viturawong/a-note-on-typescript-non-null-assertion-operator-bad8fbe62bba